task_10/dima.greshchook/First/src/main/java/com/dima/WrapperForChange.java

```diff
@@ -0,0 +1,33 @@
+package com.dima;
+
+public class WrapperForChange {
+
+  public static String changedString(String old) {
+
+    if (!isStringIncludeChar(old, '.') && !isStringIncludeChar(old, '@')){
+      return old;
+    }
+    String newString = "";
+    for (int i = 0; i < old.length(); i++) {
```

Comment:
You can also use a **replace** method instead of a cycle.

Reply:
I'm agree with you. But I've forget it. I'm going to fix this tonight. Thanks a lot.